wordpress-mobile/WordPress-Android
# Add Date Headers to "Scan History" screen

### Description
Add date headers to scan history list like the ones in activity log screen. #13992 


### To test:
1. Select a website that has scan feature
2. Tap "scan" on home screen of mobile app
3. Tap "History" at top-right of the screen
4. Verify that the date headers shown as expected

<img src="https://user-images.githubusercontent.com/46663626/107807720-eafe7a00-6d96-11eb-8446-7b424625d7cc.jpg" alt="screenshot_scan_history" width="50%"/>


#### PR submission checklist:
